### PR TITLE
[Update] add PE link

### PR DESCRIPTION
**Pull request by Icemap**
### First-time contributors' checklist 

- [ ] I've signed [**Contributor License Agreement**](https://cla-assistant.io/pingcap/docs-cn) that's required for repo owners to accept my contribution.

### Which TiDB version(s) do your changes apply to? (Required)

**Tips for choosing the affected version(s):**

By default, **CHOOSE MASTER ONLY** so your changes will be applied to the next TiDB major or minor releases. If your PR involves a product feature behavior change or a compatibility change, **CHOOSE THE AFFECTED RELEASE BRANCH(ES) AND MASTER**.

For details, see [tips for choosing the affected versions (in Chinese)](https://github.com/pingcap/docs-cn/blob/master/CONTRIBUTING.md#版本选择指南).

- [ ] master (the latest development version)
- [ ] v6.2 (TiDB 6.2 versions)
- [ ] v6.1 (TiDB 6.1 versions)
- [ ] v5.4 (TiDB 5.4 versions)
- [ ] v5.3 (TiDB 5.3 versions)
- [ ] v5.2 (TiDB 5.2 versions)
- [ ] v5.1 (TiDB 5.1 versions)
- [ ] v5.0 (TiDB 5.0 versions)

### What is the related PR or file link(s)?

- This PR is translated from:
- Other reference link(s):

### Do your changes match any of the following descriptions?

- [ ] Delete files
- [ ] Change aliases
- [ ] Need modification after applied to another branch 
- [ ] Might cause conflicts after applied to another branch

### CODE REVIEW

File: develop/dev-guide-transaction-overview.md [hunk 1]
```diff
@@ -172,3 +172,7 @@ ERROR 8048 (HY000): The isolation level 'SERIALIZABLE' is not supported. Set tid
 ```
 
 TiDB 实现了快照隔离 (Snapshot Isolation, SI) 级别的一致性。为与 MySQL 保持一致，又称其为“可重复读”。该隔离级别不同于 [ANSI 可重复读隔离级别](/transaction-isolation-levels.md#与-ansi-可重复读隔离级别的区别) 和 [MySQL 可重复读隔离级别](/transaction-isolation-levels.md#与-mysql-可重复读隔离级别的区别)。更多细节请阅读 [TiDB 事务隔离级别](/transaction-isolation-levels.md)。
+
+## 拓展阅读
+
+- PingCAP 免费在线课程 [TiDB 特有功能与事务控制 - TiDB v6](https://learn.pingcap.com/learner/course/750002/?utm_source=docs-cn-dev-guide): 了解可用于应用程序的 TiDB 独特功能，如 `AUTO_RANDOM` 及 `AUTO_INCREMENT` 特别注意事项、全局临时表、如何使用 TiFlash 启用 HTAP 以及放置策略等。
```
Comment (Oreoxmt) [inline develop/dev-guide-transaction-overview.md:178]:
```suggestion
- PingCAP 免费在线课程 [TiDB 特有功能与事务控制 - TiDB v6](https://learn.pingcap.com/learner/course/750002/?utm_source=docs-cn-dev-guide)：了解可用于应用程序的 TiDB 独特功能，如 `AUTO_RANDOM` 及 `AUTO_INCREMENT` 特别注意事项、全局临时表、如何使用 TiFlash 启用 HTAP 以及放置策略等。
```

File: develop/dev-guide-transaction-overview.md [hunk 2]
```diff
@@ -172,3 +172,7 @@ ERROR 8048 (HY000): The isolation level 'SERIALIZABLE' is not supported. Set tid
 ```
 
 TiDB 实现了快照隔离 (Snapshot Isolation, SI) 级别的一致性。为与 MySQL 保持一致，又称其为“可重复读”。该隔离级别不同于 [ANSI 可重复读隔离级别](/transaction-isolation-levels.md#与-ansi-可重复读隔离级别的区别) 和 [MySQL 可重复读隔离级别](/transaction-isolation-levels.md#与-mysql-可重复读隔离级别的区别)。更多细节请阅读 [TiDB 事务隔离级别](/transaction-isolation-levels.md)。
+
+## 拓展阅读
+
+- PingCAP 免费在线课程 [TiDB 特有功能与事务控制 - TiDB v6](https://learn.pingcap.com/learner/course/750002/?utm_source=docs-cn-dev-guide)：了解可用于应用程序的 TiDB 独特功能，如 `AUTO_RANDOM` 及 `AUTO_INCREMENT` 特别注意事项、全局临时表、如何使用 TiFlash 启用 HTAP 以及放置策略等。
```
Comment (qiancai) [inline develop/dev-guide-transaction-overview.md:178]: 可能这个位置太靠后了容易被用户错过，建议将此视频放到文档开头点的位置，参考 https://docs.pingcap.com/zh/tidb/stable/dumpling-overview

File: develop/dev-guide-transaction-overview.md [hunk 3]
```diff
@@ -172,3 +172,7 @@ ERROR 8048 (HY000): The isolation level 'SERIALIZABLE' is not supported. Set tid
 ```
 
 TiDB 实现了快照隔离 (Snapshot Isolation, SI) 级别的一致性。为与 MySQL 保持一致，又称其为“可重复读”。该隔离级别不同于 [ANSI 可重复读隔离级别](/transaction-isolation-levels.md#与-ansi-可重复读隔离级别的区别) 和 [MySQL 可重复读隔离级别](/transaction-isolation-levels.md#与-mysql-可重复读隔离级别的区别)。更多细节请阅读 [TiDB 事务隔离级别](/transaction-isolation-levels.md)。
+
+## 拓展阅读
```
Comment (Oreoxmt) [inline develop/dev-guide-transaction-overview.md:176]:
```suggestion
## 拓展学习视频
```

File: develop/dev-guide-overview.md [hunk 4]
```diff
@@ -44,3 +45,4 @@ TiDB 高度兼容 MySQL 协议，TiDB 支持[大多数 MySQL 的语法及特性]
 - [事务](/develop/dev-guide-transaction-overview.md)
 - [优化 SQL 性能](/develop/dev-guide-optimize-sql-overview.md)
 - [示例程序](/develop/dev-guide-sample-application-spring-boot.md)
+- [TiDB SQL 开发免费在线课程](https://pingcap.com/zh/courses-catalog/back-end-developer/?utm_source=docs-cn-dev-guide)
```
[the remark this answers is not in the document]
Comment (Icemap) [reply develop/dev-guide-overview.md:48]: Done

File: develop/dev-guide-transaction-overview.md (hunk 2 above)
Comment (Icemap) [reply develop/dev-guide-transaction-overview.md]: Done

File: develop/dev-guide-overview.md (hunk 4 above)
Comment (Oreoxmt) [inline develop/dev-guide-overview.md:48]: 添加到**快速开始**下面
```
- [快速开始](/develop/dev-guide-build-cluster-in-cloud.md)
- [TiDB SQL 开发免费在线课程](https://pingcap.com/zh/courses-catalog/back-end-developer/?utm_source=docs-cn-dev-guide)
...

```

File: develop/dev-guide-sample-application-java.md [hunk 5]
```diff
@@ -11,6 +11,11 @@ aliases: ['/zh/tidb/dev/sample-application-java']
...
 本文档将展示如何使用 TiDB 和 Java 来构造一个简单的 CRUD 应用程序。
 
+## PingCAP 免费在线课程
+
+- [使用 Connector/J - TiDB v6](https://learn.pingcap.com/learner/course/840002/?utm_source=docs-cn-dev-guide)
+- [在 TiDB 上开发应用的最佳实践 - TiDB v6](https://learn.pingcap.com/learner/course/780002/?utm_source=docs-cn-dev-guide)
+
 > **注意：**
 >
 > 推荐使用 Java 8 及以上版本进行 TiDB 的应用程序的编写。
```
Comment (Oreoxmt) [inline develop/dev-guide-sample-application-java.md:21]:
```suggestion
> **注意：**
>
> 推荐使用 Java 8 及以上版本进行 TiDB 的应用程序的编写。

## 拓展学习视频

- [使用 Connector/J - TiDB v6](https://learn.pingcap.com/learner/course/840002/?utm_source=docs-cn-dev-guide)
- [在 TiDB 上开发应用的最佳实践 - TiDB v6](https://learn.pingcap.com/learner/course/780002/?utm_source=docs-cn-dev-guide)

```

File: develop/dev-guide-transaction-overview.md [hunk 6]
```diff
@@ -7,6 +7,10 @@ summary: 简单介绍 TiDB 中的事务。
 
 TiDB 支持完整的分布式事务，提供[乐观事务](/optimistic-transaction.md)与[悲观事务](/pessimistic-transaction.md)（TiDB 3.0 中引入）两种事务模型。本文主要介绍涉及到事务的语句、乐观事务和悲观事务、事务的隔离级别，以及乐观事务应用端重试和错误处理。
 
+## PingCAP 免费在线课程
```
Comment (Oreoxmt) [inline develop/dev-guide-transaction-overview.md:10]:
```suggestion
## 拓展学习视频
```

File: develop/dev-guide-overview.md (hunk 4 above)
Comment (Icemap) [reply develop/dev-guide-overview.md]: Done

**Review by Oreoxmt — APPROVED** [review]
LGTM

File: develop/dev-guide-overview.md [hunk 7]
```diff
@@ -10,11 +10,14 @@ aliases: ['/zh/tidb/dev/developer-guide-overview']
...
 
+此外，你还可以通过视频的形式学习免费的 [TiDB SQL 开发在线课程](https://pingcap.com/zh/courses-catalog/back-end-developer/?utm_source=docs-cn-dev-guide)。
+
 ## TiDB 基础
 
 在你开始使用 TiDB 之前，你需要了解一些关于 TiDB 数据库的一些重要工作机制：
 
 - 阅读 [TiDB 事务概览](/transaction-overview.md)来了解 TiDB 的事务运作方式或查看[为应用开发程序员准备的事务说明](/develop/dev-guide-transaction-overview.md)查看应用开发程序员需要了解的事务部分。
+- 学习免费在线课程 [TiDB 架构与特点](https://learn.pingcap.com/learner/course/600003/?utm_source=docs-cn-dev-guide)，了解构建 TiDB 分布式数据库集群的核心组件及其概念。
 - 此外，你需要了解[应用程序与 TiDB 交互的方式](#应用程序与-tidb-交互的方式)。
```
Comment (qiancai) [inline develop/dev-guide-overview.md:21]:
```suggestion
- 了解[应用程序与 TiDB 交互的方式](#应用程序与-tidb-交互的方式)。
```

**Review by qiancai — APPROVED** [review]

**Pull request merged**